### PR TITLE
Advent Pro: Version 3.000 added

### PR DESCRIPTION
**Pull request by yanone**
018e7c3: [gftools-packager] Advent Pro: Version 3.000 added

* Advent Pro Version 3.000 taken from the upstream repo https://github.com/googlefonts/Advent at commit https://github.com/googlefonts/Advent/commit/036dd5878f3b92c39fd74def865e6b2755bba8ba.

### CODE REVIEW

**Comment by yanone** [comment]
When I was checking the fonts on my computer, it was only showing the missing character fails, otherwise nothing. 
Please pay extra attention to this upgrade, especially the naming bits. I split a VF into upright/ital and updated the name table for the Italic manually (with a script). The live static fonts don't have a italics.

File: ofl/adventpro/METADATA.pb [hunk 1]
```diff
@@ -3,70 +3,41 @@ designer: "Andreas Kalpakidis"
...
 fonts {
   name: "Advent Pro"
   style: "normal"
   weight: 400
-  filename: "AdventPro-Regular.ttf"
+  filename: "AdventPro[wdth,wght].ttf"
   post_script_name: "AdventPro-Regular"
-  full_name: "Advent Pro Regular"
-  copyright: "Copyright (c) 2008 Andreas Kalpakidis (hello@inderesting.com), with Reserved Font Name \"Advent Pro\""
+  full_name: "AdventPro Regular"
```
Comment (RosaWagner) [inline ofl/adventpro/METADATA.pb:12]: @yanone indeed this is not consistent with previous version, full name should be "Advent Pro Regular"

**Comment by RosaWagner** [comment]
@yanone when you have time you may want to check also the issue I posted in fonttools for varLib.instancer: https://github.com/fonttools/fonttools/issues/2613#issuecomment-1310480227. If you have ital=1 instead of something else, there is already less stuff to patch.

**Comment by yanone** [comment]
Updated

## Advent Pro: Version 3.000 added

---
 ff707bd: [gftools-packager] Advent Pro: Version 3.000 added

* Advent Pro Version 3.000 taken from the upstream repo https://github.com/googlefonts/Advent at commit https://github.com/googlefonts/Advent/commit/d96f8af03e060381843a08f40d73835abd5ea8df.

**Comment by yanone** [comment]
Done

**Comment by yanone** [comment]
Updated

## Advent Pro: Version 3.000 added

---
 23d6087: [gftools-packager] Advent Pro: Version 3.000 added

* Advent Pro Version 3.000 taken from the upstream repo https://github.com/googlefonts/Advent at commit https://github.com/googlefonts/Advent/commit/c60be3e6981255367ea03ade174a6adab7a29c09.

**Comment by RosaWagner** [comment]
- Italic is all good except for:

```
<namerecord nameID="25" platformID="3" platEncID="1" langID="0x409">
      AdventPro
</namerecord>
```

Should be `AdventProItalic`

- For glyphset coverage, we would need at least these:

```
0x00BA (MASCULINE ORDINAL INDICATOR)

0x00B7 (MIDDLE DOT)

0x2013 (EN DASH)

0x2014 (EM DASH)

And 0x2212 (MINUS SIGN)
```

And capital eszett would be a plus if you feel like designing it.

- I find this WARN more necessary than it seems to avoid user confusion; maybe we should make it a FAIL?

⚠ WARN Glyph '.notdef' should contain a drawing, but it is empty. [code: empty]

**Comment by yanone** [comment]
Updated

## Advent Pro: Version 3.000 added

---
 750fd23: [gftools-packager] Advent Pro: Version 3.000 added

* Advent Pro Version 3.000 taken from the upstream repo https://github.com/googlefonts/Advent at commit https://github.com/googlefonts/Advent/commit/d206a139ee9045993fbd1e530b93f28f8bf4e3b1.

File: ofl/adventpro/DESCRIPTION.en_us.html [hunk 2]
```diff
@@ -4,6 +4,9 @@
     give an edge to your typography.
 </p>
 <p>
-    To contribute to the project, please see <a
-        href="https://github.com/googlefonts/Advent">github.com/googlefonts/Advent</a>.
-</p>
\ No newline at end of file
+    The family was upgraded to a variable font with additional Italic styles in December 2023.
```
Comment (fitojb) [inline ofl/adventpro/DESCRIPTION.en_us.html]: Shouldn’t this say “December 2022” instead? =)

Comment (RosaWagner) [reply ofl/adventpro/DESCRIPTION.en_us.html]: Was corrected in previous commit 3 days ago https://github.com/google/fonts/pull/5522/commits/4e7a3bf5849bda3cafdc713623d310bbb95375b6

**Pull request merged**